bin/vpn-dns

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+echo "Monitoring for VPN DNS changes"
+
+while true; do
+  output=$(scutil << EOF
+list "State:.*VPN"
+exit
+EOF
+  )
+
+  if [[ $output =~ State:/Network/Service/(.+)/VPN ]]; then
+    uuid=${BASH_REMATCH[1]}
+
+    echo "Found ${uuid}"
+
+    output=$(scutil << EOF
+show State:/Network/Service/${uuid}/IPv4
+exit
+EOF
+  )
+
+    echo $output | egrep -o 'Addresses : <array> { [^{]*?10\.80\.81\.\d+[^{]*? }' > /dev/null
+    if [[ $? = "0" ]]; then
+      echo "Detected VPN on 10.80.81.0/24"
```

Comment:
I'm guessing this will always be the ip?

Reply:
Yeah for the foreseeable future. I needed some way to detect the right VPN key in the config store. The UUID changes at unpredictable times so I have to discover it every time.